<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_ActionsShortcutsDialog</class>
 <widget class="QDialog" name="LC_ActionsShortcutsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Action Shortcuts Mappings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../../res/icons/icons.qrc">
    <normaloff>:/icons/shortcuts_settings.svg</normaloff>:/icons/shortcuts_settings.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="gbShortcut">
     <property name="title">
      <string>Shortcut</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="lblKeySequence">
        <property name="text">
         <string>Key Sequence:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnReset">
        <property name="toolTip">
         <string>Reset shortcut for current action to it's default value</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="LC_ShortcutButton" name="btnRecord">
        <property name="toolTip">
         <string>Shortcut key sequence recording</string>
        </property>
        <property name="text">
         <string>Start Recording</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leKeySequence">
        <property name="toolTip">
         <string>Keyboard shortcut value</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="pbClear">
        <property name="toolTip">
         <string>Removes shortcut for the action</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>For</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblGroupName">
          <property name="text">
           <string>group</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblActionIcon">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblActionName">
          <property name="text">
           <string>name</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Action Shortcuts</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="LC_ShortcutsTreeView" name="tvMappingsTree">
        <property name="toolTip">
         <string>Click on action for details. Double click on action to edit. </string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pbImport">
          <property name="toolTip">
           <string>Loads shortcuts mapping from external file.</string>
          </property>
          <property name="text">
           <string>Import...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbExport">
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
          <property name="toolTip">
           <string>Export shortcuts mappings to external file</string>
          </property>
          <property name="text">
           <string>Export...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnResetAll">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>All changes made during editing sessions are discarded to initial values</string>
          </property>
          <property name="text">
           <string>Reset All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Filter:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leFilter">
          <property name="toolTip">
           <string>Filtering actions by name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbMatchHighlight">
          <property name="toolTip">
           <string>If checked, matched actions will be highlighted onlye, otherwide - non-matched actions are hidden</string>
          </property>
          <property name="text">
           <string>Highlight</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblMessage">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LC_ShortcutButton</class>
   <extends>QPushButton</extends>
   <header>../ui/dialogs/settings/shortcuts/lc_shortcutbutton.h</header>
  </customwidget>
  <customwidget>
   <class>LC_ShortcutsTreeView</class>
   <extends>QTreeView</extends>
   <header>../ui/dialogs/settings/shortcuts/lc_shortcutstreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../res/icons/icons.qrc"/>
  <include location="../../../../../res/icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LC_ActionsShortcutsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LC_ActionsShortcutsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
